hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AbfsConfiguration.java

```diff
@@ -225,11 +227,20 @@
       DefaultValue = DEFAULT_VALUE_UNKNOWN)
   private String clusterType;
 
+  @StringConfigurationValidatorAnnotation(ConfigurationKey = FS_AZURE_CLIENT_CORRELATIONID,
+          DefaultValue = EMPTY_STRING)
+  private String clientCorrelationID;
+
+  @IntegerConfigurationValidatorAnnotation(ConfigurationKey = FS_AZURE_TRACINGCONTEXT_FORMAT,
+          DefaultValue = 1)// DEFAULT_TRACINGCONTEXT_FORMATSIZE)
```

Comment:
Why is the enum value being read as index ? Direct enum value can be used. I recall my prev comment where i had a reference mentioned.

Reply:
Updated; using direct enum